src/server/routes/api/classrooms/index.js

```diff
@@ -255,25 +286,39 @@ router.post("/:classroomId/ask", async (req, res) => {
     return res.json({ success: false, message: "You are not a member of that classroom" });
   }
 
+  const updatedClassroom = await Classrooms.findById(req.params.classroomId);
+  const usersQuestions = updatedClassroom.questions.filter( q => 
+    (q.user.toString() === req.user.id.toString()) 
+    && (q.question === question)
+  );
+  const newQuestion = usersQuestions[usersQuestions.length-1];
```

Comment:
with addtoset logic there is no guarantee the question is the last one in the list. another reason to keep push.